### DISCUSSION TITLE
GitHub approved my request for student pack in 07.29, but i don't have any of the benefits.

### DISCUSSION THREAD

**Discussion by hujberhunor** [community/community#62499] [GitHub Education]
### Hi everyone,

Basically i did not use an ID, i used a "Certificate of student legal status" pdf. 

I uploaded the file 07.29 and since then no email or notification about that other that this: 
![image](https://github.com/community/community/assets/50179148/a161e2cd-5e9d-4a9b-85da-5d69c3a4bfff)
![image](https://github.com/community/community/assets/50179148/72d3a21b-d8d7-42ec-8733-d1d7bf27f669)

I can still access this page:
![image](https://github.com/community/community/assets/50179148/7cf64a95-4f04-4275-b9d5-d9ff3e732760)

And when i click on "benefits" tab it redirect me to [this](https://education.github.com/discount_requests/application)application site. 
My profile says that i have the benefit too. 
![image](https://github.com/community/community/assets/50179148/bad1e333-f751-4548-9db2-240d0f8ad25e)

I dont know what to do at this point and i really need to be labeled as a student, if im not i cant use Working Copy to sync my files.

Any idea?

**Comment by D3Dream** [comment]
4-6 weeks

**Comment by Sajjad-mv** [comment]
I have the same issue. My request was approved on 1st August, but still No Benefits!
